package.json

```diff
@@ -155,5 +155,8 @@
   "publishConfig": {
     "access": "public"
   },
-  "types": "./dist/index.d.ts"
+  "types": "./dist/index.d.ts",
+  "engines": {
+    "node": ">=12 <20.0.0"
```

Comment:
Node 12 is EOL'd, but for us to drop it, that would require a major version bump,  which we can do in a separate PR,  so this is good.  

Not really a blocker,  but my preference would be to only have ">=12" and not the "<20.0.0"